protostar/commands/format/format_command.py

```diff
@@ -36,7 +37,7 @@ def arguments(self) -> List[Command.Argument]:
             Command.Argument(
                 name="target",
                 description=("Target to format, can be a file or a directory."),
-                type="path",
+                type="str",
```

Comment:
Maybe new type is needed — `glob`.

Reply:
meh

Reply:
maybe when we're going to use it more, now it's overkill

Reply:
You could move `map_targets_to_filepaths` logic there.

Reply:
yeah i know but it's too much effort (introducing the new type, testing, refactoring) for the payoff right now. i don't think there's going to be much maintenance with what we have now, and we're probably never going to use that type anywhere

Reply:
A user doesn't know why target uses `str` type instead of `Path` type. It would make the help description and the CLI docs more explicit.

@mkaput wdyt?